Analyse.cs

```diff
@@ -55,5 +61,20 @@ public Analyse(Input input)
             }
         }
     }
-    
+
+    /// <summary>
+    /// Finds all words longer than 7 letters.
+    /// </summary>
+    /// <param name="input">The Input object to review.</param>
+    /// <returns>An array of words longer than 7 letters.</returns>
+    private List<string> GetLongWords(Input input)
+    {
+        List<string> longWords = 
+            (
+            from word in Regex.Matches(input.InputText, "[\w']{8,}", RegexOptions.IgnoreCase)
```

Comment:
Remove redundant RegexOptions argument.
```suggestion
            from word in Regex.Matches(input.InputText, "[\w']{8,}")
```